calculate_bmi/solution/calculate_bmi.erl

```diff
@@ -0,0 +1,20 @@
+-module(calculate_bmi).
+
+-export([bmi/1, classify/1]).
+
+-include("../src/person_record.hrl").
+
+bmi(P) when is_record(P, person) ->
+    P#person.weight / math:pow(P#person.height, 2).
+
+classify(P) ->
```

Comment:
if you have a function that only does a case over the arguments, you most likely should use multiple function clauses instead

Reply:
I'll change that!